spec/integration/publishing_spec.rb

```diff
@@ -12,9 +12,9 @@
 
     date = Dimensions::Date.build(Date.today)
 
-    Facts::Metric.create(dimensions_organisation: org1, dimensions_item: item1, dimensions_date: date)
-    Facts::Metric.create(dimensions_organisation: org1, dimensions_item: item2, dimensions_date: date)
-    Facts::Metric.create(dimensions_organisation: org2, dimensions_item: item3, dimensions_date: date)
+    Facts::Metric.create(dimensions_organisation: org1, dimensions_item: item1, dimensions_date: date, pageviews: 0, unique_pageviews: 0)
```

Comment:
If `pageviews` and `unique_pageviews` are not mandatory, then we just can ignore them here. Using `FactoryGirl`, and defaulting metrics, could also do the trick.

Reply:
Fixed with eaa10d4de322c29e1eee762c867989a8558b1a07